metadata/src/main/java/org/apache/kafka/controller/errors/EventHandlerExceptionInfo.java

```diff
@@ -191,6 +191,10 @@ public String failureMessage(
             }
         }
         bld.append(".");
+        if (!isFault && internalException.getMessage() != null) {
+            bld.append(" Detailed exception message: ");
```

Comment:
The "Detailed " bit seems unecessary, maybe just "Exception message: "?

Reply:
Removed "Detailed" from the message